### ISSUE TITLE
Brand Icon for Microsoft Teams?

### ISSUE THREAD

**Issue by coreyroth** [OfficeDev/office-ui-fabric-core#937]
Now that Microsoft Teams has hit GA, it would be nice to get a brand icon for it.  I can scrape an image from somewhere but the experience would be better if it came from the icon set provided by Fabric Core.

**Comment by mikewheaton** [comment]
A new icon, `TeamsLogo`, has been added in #946 and will be included when Fabric Core 7.0.0 is released.

![image](https://cloud.githubusercontent.com/assets/1382445/24718921/aedb200c-19ec-11e7-9fa4-322b27eb4cdb.png)

**Comment by Danny-Engelman** [comment]
FYI, ``Teamslogo`` and ``TeamslogoFill`` do not follow the naming convention;

should be: ``TeamsLogo`` and ``TeamsLogoFill``

**Comment by mikewheaton** [comment]
Good catch, @Danny-Engelman, I'll submit a new PR to fix this and will raise the issue with our icons team to make sure it doesn't sneak into a future release.

**Comment by mikewheaton** [comment]
Opened #951 to fix the capitalization issue.

**Comment by htryggva** [comment]
@mikewheaton I think a proper Teams Brand Icon is still needed to keep the white areas of the brand icon.

Notice how the background color bleeds through the TeamsLogo:

![image](https://user-images.githubusercontent.com/6515360/40910015-5323243c-67eb-11e8-8dc1-62c797bb25ff.png)

**Comment by mikewheaton** [comment]
Good point, @htryggva. I'll re-open this to see if we can get SVG and PNG resources added to the [Office brand icons](https://developer.microsoft.com/en-us/fabric#/styles/brand-icons).

There is a `TeamsLogoFill` icon that seems to be meant to solve this issue. You can layer it under `TeamsLogo` like so: https://codepen.io/mikewheaton/pen/qKbzzE

But it doesn't look right:
![image](https://user-images.githubusercontent.com/1382445/40928691-1d4e74d8-67d7-11e8-9995-45272972f3f3.png)
